src/FantasyFun.API/ViewModel/TeamName.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace FantasyFun.API.ViewModel
+{
+    public class TeamName
```

Comment:
I recommend to use readonly properties here (only get, not set) and create a constructor